exercises/practice/linked-list/test/linked_list_test.exs

```diff
@@ -0,0 +1,261 @@
+defmodule LinkedListTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "pop gets element from the list" do
+    linked =
+      LinkedList.new()
+      |> LinkedList.push(7)
+
+    {value, _} = LinkedList.pop(linked)
+    assert value == 7
+  end
+
+  @tag :pending
+  test "push/pop respectively add/remove at the end of the list" do
+    linked =
+      LinkedList.new()
+      |> LinkedList.push(11)
+      |> LinkedList.push(13)
+
+    {value, linked} = LinkedList.pop(linked)
+    assert value == 13
+    {value, _} = LinkedList.pop(linked)
+    assert value == 11
```

Comment:
@angelikatyborska how does the website work with multiple assertions? iirc correctly, didn't we have to limit it to one assertion per test?

Reply:
No, no, it's completely fine. The only limitation we have is that concept exercise tests need to be self-contained within the `test do ... end` block because the file won't be shown in the editor.

Reply:
ah, yes, that's the caveat